packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -0,0 +1,196 @@
+import {
+  InitPopoverEventPayload,
+  ClearPopoversEventPayload,
+} from './popover-event';
+
+/**
+ * @internal
+ * The `atomic-popover` component displays any facet as a popover menu.
+ * @part popover-button - The button to click to display or hide the popover menu.
+ * @part label - The popover button label.
+ * @part arrow-icon - The arrow icon on the button to display or hide the popover menu.
+ * @part placeholder - The placeholder displayed when the facet is loading.
+ * @part popover-wrapper - The wrapper that contains the 'popover-button' and the 'slot-wrapper'.
+ * @part slot-wrapper - The wrapper that contains the 'facet' or 'slot'.
+ */
+
+@Component({
+  tag: 'atomic-popover',
+  styleUrl: 'atomic-popover.pcss',
+  shadow: true,
+})
+export class AtomicPopover implements InitializableComponent {
+  @InitializeBindings()
+  public bindings!: Bindings;
+  public searchStatus!: SearchStatus;
+  @BindStateToController('searchStatus')
+  @State()
+  public searchStatusState!: SearchStatusState;
+  @State()
+  public error!: Error;
+
+  @State() public isMenuVisible = false;
+  @State() public facetId?: string;
+  @State() public facetLabel?: string = 'no-label';
+  @Element() host!: HTMLElement;
+  private facetElement?: HTMLElement;
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+
+    if (this.host.children.length === 0) {
+      this.error = {
+        name: 'No child in popover',
+        message: 'One child is required inside a set of popover tags',
+      };
+    } else if (this.host.children.length > 1) {
+      this.error = {
+        name: 'Too many children in popover',
+        message: 'Cannot have more than one child inside a set of popover tags',
+      };
+    }
+  }
+
+  private get popoverId() {
+    return `${this.facetId}-popover`;
+  }
+
+  popoverOpened() {
+    if (!this.isMenuVisible) {
+      const popoverOpened = new CustomEvent<ClearPopoversEventPayload>(
+        'clearPopovers',
+        {
+          detail: {ignorePopoverFacetId: this.facetId},
+        }
+      );
+      document.dispatchEvent(popoverOpened);
+    }
+
+    this.isMenuVisible = !this.isMenuVisible;
+  }
+
+  @Listen('clearPopovers', {target: 'document'})
+  popOpened(event: CustomEvent<ClearPopoversEventPayload>) {
+    if (event.detail.ignorePopoverFacetId !== this.facetId) {
+      this.isMenuVisible = false;
+    }
+  }
+
+  renderValueButton() {
+    return (
+      <Button
+        style="square-neutral"
+        onClick={() => this.popoverOpened()}
+        part="label-button"
+        ariaExpanded={`${this.isMenuVisible}`}
+        ariaLabel={`Pop-up filter on ${this.facetLabel} facet`}
+        ariaControls={this.popoverId}
+        class={`rounded flex box-border h-full items-center mr-1.5 p-2.5 group ${
+          this.isMenuVisible
+            ? 'selected border-primary ring ring-ring-primary-light text-primary'
```

Comment:
related to css comment
```suggestion
            ? 'block border-primary ring ring-ring-primary-light text-primary'
```

Reply:
actually we can remove it the selected tag